src/store/store.js

```diff
@@ -20,7 +21,9 @@ const state = {
 const getters = {
   // eslint-disable-next-line
   srcUrl(state) {
+    console.log(24, state.schema, state.activityIndex);
     if (!_.isEmpty(state.schema) && state.activityIndex) {
```

Comment:
if state.activityIndex is 0, the if statement doesn't evaluate to 'true'. Do '... && state.activityIndex >=0`